optout/src/main/java/gov/cms/ab2d/optout/OptOutConstants.java

```diff
@@ -22,7 +22,7 @@ public class OptOutConstants {
     public static final String CONF_FILE_NAME = "#EFT.ON.AB2D.NGD.CONF.";
     public static final String CONF_FILE_NAME_PATTERN = "'D'yyMMdd.'T'HHmmsss";
     public static final String UPDATE_STATEMENT = "UPDATE public.current_mbi\n" +
-            "SET opt_out_flag = ?, effective_date = current_date\n" +
+            "SET share_data = ?, effective_date = current_date\n" +
             "WHERE mbi = ?";
 
     private OptOutConstants() {
```

Comment:
Looks like you local branch is not up to date.
COUNT_STATEMENT also should be updated: https://github.com/CMSgov/ab2d-lambdas/blob/main/optout/src/main/java/gov/cms/ab2d/optout/OptOutConstants.java#L28

Reply:
Good catch Anna.. Rebased and Updated